OrbitGl/App.cpp

```diff
@@ -1175,8 +1172,6 @@ DataView* OrbitApp::GetOrCreateDataView(DataViewType type) {
 //-----------------------------------------------------------------------------
 void OrbitApp::InitializeClientTransactions() {
   transaction_client_ = std::make_unique<TransactionClient>(GTcpClient.get());
-  symbols_client_ =
```

Comment:
This was the last "Service"/"Client" to use `TransactionService`, `TransactionClient`, right? Could you remove those as well and all related initialization code?

Reply:
Yes, there is another PR already for that #715